xblock/test/test_mixins.py

```diff
@@ -204,3 +214,19 @@ def test_add_xml_to_node(self):
                 'unique_id_default_with_force_export': 'Value5',
             }
         )
+
+    @ddt.data(
+        (None, {'datetime': ''}),
+        (datetime(2014, 4, 1, 2, 3, 4, 567890).replace(tzinfo=pytz.utc), {'datetime': '2014-04-01T02:03:04.567890'})
+    )
+    @ddt.unpack
+    def test_datetime_serialization(self, string_value, expected_attributes):
```

Comment:
@e-kolpakov Since the second parameter of this method gets a `datetime` object (or `None`) from `ddt`, calling it `string_value` is slightly unintuitive. Maybe rename it to something like `datetime_value`, or even just `value`?